### ISSUE TITLE
Porting to Aurelia.

### ISSUE THREAD

**Issue by ferndot** [codexa/firetext#356]
I am considering porting the app to Aurelia + ESNext.

That will greatly simplify UI management and module management and also enable us to utilize the new functions in ESNext.

What do you think?

**Comment by Ryuno-Ki** [comment]
I am following Aurelia from the beginning (i.e. after the renaming from Durandal).

In the beginning, the setup was quite hard (with System.js and stuff), but they released a CLI, which should ease things. On the other hand, when a framework needs a CLI it sounds way to complicated to me …

However, I really like, how Aurelia gets out of your way and let you write almost Vanilla JS. Not comparable with Vue, React or Angular!

I plan to dig deeper into it. Or into Svelte.

**Comment by ferndot** [comment]
@Ryuno-Ki 

Yeah it certainly has an interesting setup process. I really like the simplicity of working with views and the bidirectional binding.

Svelte looks like a better implementation of the jQuery API? Or is there something I am missing?

**Comment by Ryuno-Ki** [comment]
Svelte is attempting to get rid of runtime frameworks by compiling everything ahead-of-time and spitting out Vanilla JS.
So you get the best of both worlds.
But you have to write code in a way that is statically analysable. The community is welcoming on Gitter.

**Comment by ferndot** [comment]
@Ryuno-Ki thanks I will check it out!

**Comment by Ryuno-Ki** [comment]
Lately, Aurelia [teasered a book on their blog](http://blog.aurelia.io/2017/11/12/projects-that-make-aurelia-shine/) about projects, which are a good fit for it. One is „An office style application (such as Google Docs)”, which could fit to firetext quote well. I haven't ordered the book (yet).

Please blog about your experience with it :)

**Comment by ferndot** [comment]
@Ryuno-Ki sure thing!
